cmd/postgres.go

```diff
@@ -432,6 +453,80 @@ func (c *config) postgresCreateStandby() error {
 	return output.New().Print(response.Payload)
 }
 
+func (c *config) postgresPromoteToPrimary(args []string) error {
+	id, err := c.postgresID("promote-to-primary", args)
+	if err != nil {
+		return err
+	}
+
+	params := database.NewGetPostgresParams().WithID(id)
+	resp, err := c.cloud.Database.GetPostgres(params, nil)
+	if err != nil {
+		return err
+	}
+	current := resp.Payload
+
+	// copy the (minimum) current config
+	body := &models.V1PostgresUpdateRequest{
+		ProjectID:  current.ProjectID,
+		ID:         current.ID,
+		Connection: current.Connection,
+	}
+
+	// promote to primary
+	if body.Connection != nil {
```

Comment:
If connection == nil the update makes no sense right ?

Reply:
No, makes no sense. I thought about returning early here. but didn't want to return an error (because nothing broke).

But on second thought, this indeed would be a bad request, and updating the custom ressource even though the requested change is skipped will probably cause more harm than good.

Reply:
maybe:
```go
if body.Connection == nil {
   return fmt.Errorf("database has no connection, promotion makes no sense")
}
```

Reply:
I settled for this:

```
 ~ $ c postgres promote-to-primary 3bd11ccd-57f5-428f-bce5-02e6981867d9
Error: standalone postgres cluster detected, cannot be promoted to primary
```